src/Umbraco.Web.UI.Client/src/views/propertyeditors/rte/rte.html

```diff
@@ -1,6 +1,7 @@
-<div ng-controller="Umbraco.PropertyEditors.RTEController" class="umb-property-editor umb-rte">
+<div ng-controller="Umbraco.PropertyEditors.RTEController" class="umb-property-editor umb-rte" ng-class="{'--initialized': !isLoading}">
     <umb-load-indicator ng-if="isLoading"></umb-load-indicator>
 
-    <div ng-style="{ visibility :  isLoading ? 'hidden' : 'visible', width :containerWidth, height: containerHeight, overflow: containerOverflow}"
-         id="{{textAreaHtmlId}}" class="umb-rte-editor"></div>
+    <div class="umb-rte-editor-con">
```

Comment:
Why have we dropped the configurable height & width of the RTE and the containerOverflow property

As some people set a RTE to help try & match the rough size that it will take up, on the rendering page to help see where the text will flow.

Reply:
Thats a mistake, reverting it now.